index.rst

```diff
@@ -318,56 +318,101 @@ being performed to clarify the classifications of data and services together.
++------------------------+------------------------+------------+-------------------------+
+| File/Workspace Access  | read:workspace/user,   | Yes        | ws_usr                  |
+| (User/Shared)          | write:workspace/user   |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| Portal                 | exec:portal            | No         | portal                  |
++------------------------+------------------------+------------+-------------------------+
+| Notebook               | exec:notebook          | No         | nb                      |
++------------------------+------------------------+------------+-------------------------+
+
 Tokens
 ======
 
 Broadly speaking, there are two main types of tokens in the LSST DM system. Tokens whose primary use
-are for identity, which are issued from CILogon, and tokens whose primary use are for checking
-capabilities. Identity tokens are roughly equivalent to X.509 certificates; they include information
-about the user identity, including the username for the LSST account and/or the UNIX UID, and group
-memberships, in addition to a cryptographic signature for verifying the token integrity using public
-key encryption.
+are for identity, which are similar to those issued from CILogon, and tokens whose primary use are
+for checking capabilities. Identity tokens are roughly equivalent to X.509 certificates; they
+include information about the user identity, including the username for the LSST account and/or
+the UNIX UID, and group memberships, in addition to a cryptographic signature for verifying the
+token integrity using public key encryption.
 
 Capability tokens, in the LSST DM system, will minimally also include the UNIX UID and/or username
-for the LSST account, as well as a list of capabilities for the token.
+for the LSST account, as well as a list of capabilities for the token. Those capabilities are
+listed in the ``scope`` claim of a the token.
 
-Due to the additional infrastructure and definitions required for implementing capabilities-based
-authorization, we intend to implement authentication and authorization in the LSST DM system in two
-phases.
+Approaches to Authorization
+---------------------------
 
-Phased Approach to Authorization
---------------------------------
+Approach 1 is authorization primarily through identity. LSP services will rely on identity from
+identity tokens, including UID and group membership, to authorize access to services; services,
+notably the LSP API aspect, will implement impersonation in some form to delegate authorization
+to the underlying systems.
+
+Approach 2 is the implementation of authorization first through capabilities at the service level;
+followed by the same identity-based authorization techniques from Approach 1. It can be layered
+on top of Approach 1, and as a result, gradually implemented.
```

Comment:
"Note that we do not anticipate taking a capability-only approach to authorization, with no identity provided at all.  This differs slightly from the SciTokens approach, which prefers detailed, path-based capabilities and provides identity solely for informational purposes."

Reply:
Note added below these lines.